plugins/org.eclipse.fordiac.ide.model.commands/src/org/eclipse/fordiac/ide/model/commands/create/AbstractConnectionCreateCommand.java

```diff
@@ -263,16 +273,10 @@ public void setPerformMappingCheck(final boolean performMappingCheck) {
 
 	public void setVisible(final boolean visible) {
 		this.visible = visible;
-		if (null != mirroredConnection) {
-			mirroredConnection.setVisible(visible);
-		}
 	}
 
-	public void setNegated(final boolean negated) {
-		this.negated = negated;
-		if (null != mirroredConnection) {
-			mirroredConnection.setNegated(negated);
-		}
+	public void setAttributes(final List<Attribute> attributes) {
+		this.attributes.addAll(attributes);
```

Comment:
somehow also the mirrored connection needs to get the attributes

I guess this should be done in line 224